config/production.json

```diff
@@ -77,8 +77,6 @@
 		"layout/query-selected-editor": true,
 		"legal-updates-banner": false,
 		"login/magic-login": true,
-		"login/native-login-links": true,
```

Comment:
That's a good point, I forgot about Jetpack Cloud 🙂  But I now verified that the `login` helper is not used in any Jetpack Cloud code.

For OAuth-enabled environments, we'll need to carefully inspect where we are redirecting in "redirect to login if logged out" scenarios. Instead of `/log-in`, it's more proper to redirect to the `wordpress.com/oauth2/authorize` page. But that can be done in a different PR.